modules/ROOT/partials/beta-disclaimer.adoc

```diff
@@ -0,0 +1,13 @@
+== Beta Release
+
+// tag::admonition[]
+[CAUTION]
+--
+This is a pre-release (Beta) version of Couchbase Server 6.5 and is intended for development purposes only.
+We consider Beta releases to have some rough egdes and bugs, and may change significantly before the final GA release. 
+
+This release is not supported in production. 
+
+No Enterprise Support is provided for pre-release (Developer Preview and Beta) versions of the software.
```

Comment:
Should we include a statement about appreciating feedback and how to provide it?

Reply:
I don't have the details, hence not included. Will update this information later.